internal/pkg/plugin/helminstaller/helminstaller.go

```diff
@@ -0,0 +1,73 @@
+package helminstaller
+
+import (
+	"os"
+	"strings"
+
+	"github.com/devstream-io/devstream/internal/pkg/configmanager"
+	"github.com/devstream-io/devstream/internal/pkg/plugin/helminstaller/defaults"
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller/helm"
+	"github.com/devstream-io/devstream/pkg/util/log"
+	"github.com/devstream-io/devstream/pkg/util/types"
+)
+
+func GetDefaultOptionsByInstanceID(instanceID string) *helm.Options {
+	for name, options := range defaults.DefaultOptionsMap {
+		if strings.Contains(instanceID, name) {
+			return options
+		}
+	}
+	return nil
+}
+
+func SetDefaultConfig(options configmanager.RawOptions) (configmanager.RawOptions, error) {
+	helmOptions, err := helm.NewOptions(options)
+	if err != nil {
+		return nil, err
+	}
+
+	instanceID := helmOptions.InstanceID
+	defaultHelmOptions := GetDefaultOptionsByInstanceID(instanceID)
+
+	if defaultHelmOptions == nil {
+		log.Debugf("Default config for %s wasn't found.", instanceID)
+		return options, nil
+	}
+
+	log.Infof("Filling default config with instance: %s.", instanceID)
+	helmOptions.FillDefaultValue(defaultHelmOptions)
+	log.Debugf("Options with default config filled: %v.", helmOptions)
+
+	return types.EncodeStruct(helmOptions)
+}
+
+// RenderValuesYaml renders options.valuesYaml to options.chart.valuesYaml;
+// If options.valuesYaml don't contains ":", it should be a path like "./values.yaml", then read it and transfer to options.chart.valuesYaml
+func RenderValuesYaml(options configmanager.RawOptions) (configmanager.RawOptions, error) {
+	helmOptions, err := helm.NewOptions(options)
+	if err != nil {
+		return nil, err
+	}
+
+	// 1. valuesYaml don't be set
```

Comment:
don't be set -> isn't be set

Reply:
is not set haha~

we are all wrong here.